### PR TITLE
Fix material UI

### PR DESCRIPTION
**Pull request by AndreasLc1103**
To fix broken install issue and update material dependencies

- [ ] No discrepancies across browsers. Eg. chrome vs safari
- [ ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ ] CSS (https://jigsaw.w3.org/css-validator/)
- [ ] Google sign-in is operational
- [ ] Accessibility functions
- [ ] Pages can scale without distorting page
- [ ] No dead links
- [ ] Navbar is consistent across the site
- [ ] Pages are screen-reader accessible
- [ ] Contrast meets standards for accessibility
- [ ] Pages are keyboard accessible
- [ ] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

File: client/src/components/exercise/lab3/helpers/StickyFooter.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-import CssBaseline from "@material-ui/core/CssBaseline";
-import Typography from "@material-ui/core/Typography";
-import { makeStyles } from "@material-ui/core/styles";
-import Container from "@material-ui/core/Container";
-import Divider from "@material-ui/core/Divider";
+import CssBaseline from "@mui/core/CssBaseline";
+import Typography from "@mui/core/Typography";
+import { makeStyles } from "@mui/core/styles";
+import Container from "@mui/core/Container";
+import Divider from "@mui/core/Divider";
```
Comment (heathermoses) [inline client/src/components/exercise/lab3/helpers/StickyFooter.js]: I believe these should import from `@mui/material`

File: client/src/components/exercise/lab4/components/CodeUpdateHeader.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+import { Typography } from "@mui/core";
+import AppBar from "@mui/core/AppBar";
+import Toolbar from "@mui/core/Toolbar";
+import Grid from "@mui/core/Grid";
+import DialogTitle from "@mui/core/DialogTitle";
+import DialogContent from "@mui/core/DialogContent";
+import DialogContentText from "@mui/core/DialogContentText";
+import DialogActions from "@mui/core/DialogActions";
+import Button from "@mui/core/Button";
+import Dialog from "@mui/core/Dialog";
```
Comment (heathermoses) [inline client/src/components/exercise/lab4/components/CodeUpdateHeader.js]: These should import from `@mui/material`

File: client/src/components/exercise/lab4/components/Header.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import React from "react";
-import Toolbar from "@material-ui/core/Toolbar";
-import Grid from "@material-ui/core/Grid";
+import Toolbar from "@mui/core/Toolbar";
+import Grid from "@mui/core/Grid";
```
Comment (heathermoses) [inline client/src/components/exercise/lab4/components/Header.js]: I think these should also import from `@mui/material`

**Review by heathermoses — CHANGES_REQUESTED** [review]
I tested this by pulling this branch and doing an `npm install` and the project worked for me. There's a few small issues with imports, but not breaking. Looks good to me otherwise

**Review by heathermoses — APPROVED** [review]
LGTM

**Review by dm9965 — APPROVED** [review]
LGTM

**Pull request merged**